modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx

```diff
@@ -52,6 +52,10 @@ const SelectSubscription = ({
 
 	const navigate = useNavigate();
 	const {state} = useLocation();
+	const [
+		totalOfActivationKeysAvailable,
+		setTotalOfActivationKeysAvailable,
```

Comment:
can we rename these `availableActivationKeysTotal` and `setAvailableActivationKeysTotal`, that will get rid of the `Of` in the name and make it a little shorter and precise.